guide/onboarding/getting-to-know-your-users.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: guide
+title: Getting to know your users
+description: How best to understand and develop knowledge about your users.
+parent: Onboarding
+nav_order: 1
+permalink: /guide/onboarding/getting-to-know-your-users/
+main_classes: -no-top-padding
+---
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/getting-to-know-your-users.svg"
+   alt-text = "Bitcoin, key and lock"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Getting to know your users
+
+When building onboarding experiences, it is important to understand and develop knowledge about your users.
+Within the Bitcoin ecosystem, there is a wide range in the level of expertise. Identifying who you are building for is crucial, as these user groups value and understand varying characteristics of coin management tools. For example, “rookies” often favor convenient, easy-to-use wallets, while well-versed bitcoin owners tend to prioritize security[^1].
+
+## Observe your potential users
+This observation could take the form of user research, user interviews and surveys aimed at a range of participants and observe what they do, how they think, and what they want. Let's consider some things we might want to ask ourselves when improving the onboarding experience for new users:
+
+* What is their perception of Bitcoin?[^2]
+* Why do they intend to use Bitcoin, investment, savings or spending?
+* Where do they prefer to store most of their value, mobile or desktop wallets?
+* Where do they experience frustration?
+* Where does the first experience take place?
+* How did they hear about my application? Word of mouth? App store search?
+* Do first-time users have specific needs or goals?
+
+Perhaps after research, you identify your users as "cypherpunks" with sound knowledge of Bitcoin, but they are new to your app. You might build an experience that preserves as much of their privacy as possible and is super customizable from the outset.
+
+Aim to gather as much information and observations as possible in order to empathize with your users.
+
+## Identify barriers and motivators they have
+You can then use this research to identify where potential problems may exist. This will then allow you to highlight opportunities for innovation.
+
+### Rookie example
+
+Let's use an example of saving for the future and see how we can understand a user with this goal. For the sake of simplicity let's focus on one persona or maybe more accurately personal story.
```

Comment:
> For the sake of simplicity let's focus on one persona or maybe more accurately personal story.

I initially read this as: `For the sake of simplicity let's focus on one persona or [focus on a] maybe more accurately personal story.` I think it needs a comma or some other slight restructuring to make it clear that the second part of the sentence refers to the word `persona`.

Reply:
🚧